ReactSkia/platform/linux/RuntimeEventBeat.cpp

```diff
@@ -9,7 +9,7 @@
 #include "RuntimeEventBeat.h"
 #include "ReactSkia/utils/RnsLog.h"
 
-#define BEAT_INTERVAL  1000 /*unit ms. Beat interval Set to:1 sec as temp fix*/
+#define BEAT_INTERVAL  10 /*unit ms. Beat interval Set to:1 sec as temp fix*/
```

Comment:
Need to update the comment of this line. 
It will good to mention on why 10ms is set. It will be helpful for reference if this value requires change in future

Reply:
Please also check implemented events of image component and onLayout event as regression cases for this change

Reply:
sure